api/v1/env.go

```diff
@@ -302,6 +301,14 @@ func (s *Store) GetEnv() []corev1.EnvVar {
 			Name:  "SQL_SET_DEFAULT_SESSION_VARIABLES",
 			Value: "0",
 		},
+		{
+			Name:  "INSTALL_LOCALE",
+			Value: s.Spec.ShopConfiguration.Locale,
```

Comment:
I guess the key for the helm chart is `store.locale`?

Reply:
For the crd it is: `store.shopConfiguration.locale`